app/assets/javascripts/discourse/app/modifiers/draggable.js

```diff
@@ -29,8 +29,6 @@ export default class DraggableModifier extends Modifier {
 
   @bind
   dragMove(e) {
-    e.stopPropagation();
-    e.preventDefault();
```

Comment:
That has to be removed. Before, it was completely blocking scroll. Decision if scroll should be blocked or not was moved into dragMove function. In sidebar link, we stop scrolling when link is hold for 300ms.